lib/bootstrap.js

```diff
@@ -5,6 +5,7 @@ const cmdArgs = require('command-line-args');
 require('./utils/process-log');
 
 const isHelpFlagPresent = args => args.find(arg => ['help', '--help'].indexOf(arg) !== -1);
```

Comment:
we could generalize this func

Reply:
You could use `includes` instead of `indexOf`